doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -1,3 +1,91 @@
+
+* The ``integrate motion`` parameter control if the dynamics equations of the particles are calculated. If this parameter is set to false, the dynamics of the particle will not be calculated. The particle's position and velocity will only be defined by the particles' position and velocity function.
+
+* The ``particle nonlinear tolerance`` parameter controls particle dynamics' nonlinear tolerance. The nonlinear solver won't have converged until the residual on the dynamics equations of all the particles is smaller than this threshold.
+
+* The ``alpha`` parameter is the relaxation parameter used when solving the dynamics equation of the particle.
+
+* The ``fluid density`` parameter is the fluid density used in the force calculation of the resolved CFD-DEM. This parameter is redundant with others in the solver and will be removed in the upcoming code modification.
+
+* The subsection ``gravity`` defines the value of the gravity used in the simulation. This gravity can be defined as a function that evolved in time and space. Each component of the ``Function expression`` corresponds respectively to its magnitude in X, Y, and Z.
+
+* The subsection ``particle info 0`` is used to define relevant information that is specific to the particle with id 0. For each particle with the index n, a new subsection name ``particle info n`` should be defined with relevant information.
+
+The following parameter and subsection are all inside the subsection ``particle info 0`` and have to be redefined for all particle separatly.
+
+* The subsection ``position`` defines the initial value of the particle position if the parameter ``integrate motion`` is set to true. Otherwise, it defines the particle's position at all points in time. This position is expressed as a function that can evolve in time. Each component of the ``Function expression`` corresponds to the value of coordinate X, Y, and Z. 
+
+* The subsection ``velocity`` defines the initial value of the particle velocity if the parameter ``integrate motion`` is set to true. Otherwise, it defines the particle's velocity at all points in time. This velocity is expressed as a function that can evolve in time. Each component of the ``Function expression`` corresponds to the value of its component in the X, Y, and Z direction.
+
+* The subsection ``omega`` defines the initial value of the particle rotational velocity if the parameter ``integrate motion`` is set to true. Otherwise, it defines the particle's rotational velocity at all points in time. This rotational velocity is expressed as a function that can evolve in time. Each component of the ``Function expression`` corresponds to the value of its component in the X, Y, and Z direction. It's important to note that even the 2D solver uses the rotational velocity in 3D. In that case, it will only use the Z component of the rotational velocity.
```

Comment:
```suggestion
* The subsection ``omega`` defines the initial value of the particle rotational velocity if the parameter ``integrate motion=true``. Otherwise, it defines the particle's rotational velocity at all times. This rotational velocity is expressed as a function that can evolve in time. Each component of the ``Function expression`` corresponds to the value of its component in the X, Y, and Z direction. It's important to note that even the 2D solver uses the rotational velocity in 3D. In that case, it will only use the Z component of the rotational velocity.
```